BookingsApi/BookingsApi.Domain/Hearing.cs

```diff
@@ -206,7 +206,8 @@ public Participant AddJudge(Person person, HearingRole hearingRole, CaseRole cas
             Participant participant = new Judge(person, hearingRole, caseRole)
             {
                 DisplayName = displayName,
-                CreatedBy = CreatedBy
+                CreatedBy = CreatedBy,
+                Discriminator = "Judge"
```

Comment:
The discriminator is used by EF, why did you add in in the hearing class?

Reply:
I use the Dicriminator to check if the participant is Judge. Otherwise I need to include the UserRole for the participant

Reply:
That is the correct way to do it. If you really don't want to pull in the roles than have you considered

`
```suggestion
               "Judge"hearing.GetParticipants().OfType<Judge>().FirstOrDefault()
```

This will filter the list to only get judges, which typically have one or none

Reply:
I have now included the relative objects

Reply:
removed discriminator